src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEReader.cs

```diff
@@ -690,11 +690,13 @@ public bool TryOpenAssociatedPortablePdb(string peImagePath, Func<string, Stream
 
             // First try .pdb file specified in CodeView data (we prefer .pdb file on disk over embedded PDB
             // since embedded PDB needs decompression which is less efficient than memory-mapping the file).
-            var codeViewEntry = entries.FirstOrDefault(e => e.IsPortableCodeView);
```

Comment:
The .ni pdbs are not portable code view. The existing check should be sufficient to filter them out.

Reply:
I see. Thanks!